vnext/Microsoft.ReactNative/Fabric/Composition/CompositionDynamicAutomationProvider.cpp

```diff
@@ -158,6 +158,13 @@ HRESULT __stdcall CompositionDynamicAutomationProvider::GetPatternProvider(PATTE
     AddRef();
   }
 
+  if (patternId == UIA_RangeValuePatternId) {
+    *pRetVal = static_cast<IRangeValueProvider *>(this);
```

Comment:
We should probably only return this pattern if the role is set to something that aligns with this pattern.

Reply:
Shouldn't we also do a role check for UIA_ValuePatternId as well (i.e if the role is not 'adjustable' or 'progressbar' since those roles correspond to RangeValuePattern?

Reply:
Yea i think we'll want roles checks for each of these!